package.json

```diff
@@ -5,9 +5,9 @@
   "main": "dist/index.js",
   "scripts": {
     "prestart": "npm install",
-    "start": "webpack-dev-server --progress",
+    "start": "node_modules/.bin/webpack-dev-server --progress",
```

Comment:
What, no!

this should be in node_modules if you do not have webpack installed it will still run this way.
never use anything global in package.json.